ui/src/app/pages/profile/demographic-survey-panel.tsx

```diff
@@ -1,62 +1,43 @@
 import * as React from 'react';
 
-import { Button, ButtonWithLocationState } from 'app/components/buttons';
+import { ButtonWithLocationState } from 'app/components/buttons';
 import { DEMOGRAPHIC_SURVEY_V2_PATH } from 'app/utils/constants';
 import { displayDateWithoutHours } from 'app/utils/dates';
-import { serverConfigStore } from 'app/utils/stores';
 
 import { styles } from './profile-styles';
 
 interface Props {
   demographicSurveyCompletionTime: number;
-  firstSignInTime: number;
   onClick: () => void;
```

Comment:
Is this needed?

Reply:
good catch.  I can actually remove a lot more - like the whole v1 Demo Survey